.gitignore

```diff
@@ -25,3 +25,4 @@ xcuserdata
 /platform/ios/benchmark/assets/tiles/mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v6
 **/token
 /platform/macos/macos.xcworkspace/xcshareddata/macos.xcscmblueprint
+/platform/ios/pkg
```

Comment:
What about /platform/macos/pkg? It doesn’t look like the `OUTPUT` variable in platform/macos/scripts/package.sh has been changed from `build/macos/pkg` to `platform/macos/pkg` as for iOS.